bug(vllm_performance): wrong import path in breaks tests

When running the non-locked tests we get the following error:

```
_______________ ERROR collecting tests/test_vllm_otlp_traces.py ________________
ImportError while importing test module '/workspace/app/ado/plugins/actuators/vllm_performance/tests/test_vllm_otlp_traces.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
.tox/py313-nonlocked-linux/lib/python3.13/site-packages/_pytest/python.py:507: in importtestmodule
    mod = import_path(
.tox/py313-nonlocked-linux/lib/python3.13/site-packages/_pytest/pathlib.py:587: in import_path
    importlib.import_module(module_name)
/root/.local/share/uv/python/cpython-3.13.13-linux-x86_64-gnu/lib/python3.13/importlib/__init__.py:88: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
<frozen importlib._bootstrap>:1395: in _gcd_import
    ???
<frozen importlib._bootstrap>:1360: in _find_and_load
    ???
<frozen importlib._bootstrap>:1331: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:935: in _load_unlocked
    ???
.tox/py313-nonlocked-linux/lib/python3.13/site-packages/_pytest/assertion/rewrite.py:197: in exec_module
    exec(co, module.__dict__)
plugins/actuators/vllm_performance/tests/test_vllm_otlp_traces.py:10: in <module>
    from plugins.actuators.vllm_performance.ado_actuators.vllm_performance.actuator_parameters import (
E   ModuleNotFoundError: No module named 'plugins'
```

The issue is here: https://github.com/IBM/ado/blob/main/plugins/actuators/vllm_performance/tests/test_vllm_otlp_traces.py#L10-L15

The import path is wrong

--- a/plugins/actuators/vllm_performance/tests/test_vllm_otlp_traces.py
+++ b/plugins/actuators/vllm_performance/tests/test_vllm_otlp_traces.py
@@ -6,13 +6,14 @@ Unit tests for OTLP traces endpoint feature in vllm_performance actuator.
 Tests parameter validation, YAML generation, and backward compatibility.
 """
 
-from orchestrator.core.actuatorconfiguration.config import ActuatorConfiguration
-from plugins.actuators.vllm_performance.ado_actuators.vllm_performance.actuator_parameters import (
+from ado_actuators.vllm_performance.actuator_parameters import (
     VLLMPerformanceTestParameters,
 )
-from plugins.actuators.vllm_performance.ado_actuators.vllm_performance.k8s.yaml_support.build_components import (
+from ado_actuators.vllm_performance.k8s.yaml_support.build_components import (
     ComponentsYaml,
 )
+
+from orchestrator.core.actuatorconfiguration.config import ActuatorConfiguration
 
 
 class TestOTLPTracesEndpointParameter:

